docs/develop/api-reference/index.mdx

```diff
@@ -196,6 +196,96 @@ puts client.caches.to_a.join(", ")
   cli={`momento cache list`}
 />
 
+### Flush cache
+
+Flushes all data from a cache
+
+Attributes:
+
+| Name      | Type   | Description                      |
+| --------- | ------ | -------------------------------- |
+| cacheName | String | Name of the cache to be flushed. |
+
+:::tip
+
+While you could use [Delete Cache](#delete-cache), then [Create Cache](#create-cache) to mimic this, the FlushCache API keeps the settings and only deletes the data in the cache.
+
+:::
+
+<SdkExamples
```

Comment:
we should figure out how we want to validate that these still work since most of these SDKs aren't 1.0 yet

Reply:
I am all ears on how we can pay down this technical debt. Should this stop this PR from moving forward?

Reply:
Removed the code examples for now.